### ISSUE TITLE
Error python 3.6 with utf8 string

### ISSUE THREAD

**Issue by nguyenvanquan7826** [judge0/api#69]
Hi, I'm test in python 3.6 with utf8 but api return wrong. I want replace utf8 char by accsi char. Eg with "thầu ao" -> "thao ao" but your api return "thầu ao" (same input). I also test on ideone.com but this website return true (return "thao ao").

input: thầu ao
My code:
```
t1 = input();
s = ''
for c in t1:
	if c == 'ầ':
		s += 'a'
	else:
		s += c
print(s)
```
In ideone.com
https://ideone.com/XsXhFq

**Comment by hermanzdosilovic** [comment]
Hi @nguyenvanquan7826,

this is now fixed with [Judge0 API v1.2.0](https://github.com/judge0/api/releases/tag/v1.2.0).

**Comment by nguyenvanquan7826** [comment]
Thank you. I see it.
